lessons/cryptography/learn-to-count/learn-to-count.md

```diff
@@ -0,0 +1,105 @@
+# [Cryptography] Learn to Count
+*Learn about why there are different number systems.*
+
+![banner alt text](.rsrc/banner.png)
+
+# Objectives
+- Learn the following number systems: decimal, binary, hexadecimal
+
+# Introduction
+When we think of numbers, we are usually thinking about the decimal number system. This system that we are used to uses 10 as a base, and the digits range from 0 to 9 (total of 10 digits). There are also other number systems, such as binary and hexadecimal. The binary number system uses 2 as a base, and the digits range from 0 to 1 (total of 2 digits). For example, the decimal number 42 would be 0b101010 in binary. The hexadecimal number system uses 16 as a base, and the digits range from 0 to 9, and then A-F (total of 16 digits). For example, the decimal number 42 would be 0x2A in hexadicimal. 
+
+In this lesson, we will cover the following 3 decimal number systems, and why they are important. 
+
+1. Decimal
+2. Binary
+3. Hexadecimal
+
+# Lesson
+## Decimal
+
+When we first learn to count and learn simple mathematics, we use the decimal number system. We first counted with digits 0 to 9, and then learned about what to do with overflowing digits into another column. For example, the decimal number 129 can be broken down into the units, tens, and hundreds places. In the image below, you can see each place being broken down into powers of 10 at the top to represent each column. 
+
+![Decimal Number Broken Down (http://www.steves-internet-guide.com/binary-numbers-explained/)](.rsrc/decimal-number-example.jpg)
+
+129 is the same as 100 + 20 + 9, which can be derived from the top representation of each column and the number in each place. Broken down further, 129 can be represented as 1*10^2 + 2*10^1 + 9*10^0. Any decimal number can be broken down in this method. Notice that every column represents a power of 10, which is why we say that the decimal number system uses 10 as a base. 
+
+The decimal number system is commonly used in many fields, and is the number system that humans understand the best. However, many computer applications use other number systems such as binary and hexadecimal. 
+
+## Binary
+At the lowest levels of the computer, everything is represented by an eletric signal that is either on or off. This can be represented with binary numbers where **1** means **ON** and **0** means **OFF**. In order for the computer to intrepret more complicated data, the computer must encode it into binary data. 
```

Comment:
I want to highlight this to the highest degree, this is a super super critical point. Well put.

Reply:
Should I highlight this again somewhere else? Not sure if it'll make the intro too long to add it there too, but leaning towards yes it will get too long